extensions/guacamole-auth-cas/src/main/java/org/apache/guacamole/auth/cas/ticket/TicketValidationService.java

```diff
@@ -57,21 +57,21 @@
     private ConfigurationService confService;
 
     /**
-     * Validates and parses the given ID ticket, returning the Credentials object
-     * derived from the parameters provided by the CAS server in the ticket.  If the
+     * Validates and parses the given ID ticket, returning the username
+     * provided by the CAS server in the ticket.  If the
      * ticket is invalid an exception is thrown.
      *
      * @param ticket
      *     The ID ticket to validate and parse.
      *
      * @return
-     *     The Credentials object derived from parameters provided in the ticket.
+     *     The username derived from the ticket.
      *
      * @throws GuacamoleException
      *     If the ID ticket is not valid or guacamole.properties could
      *     not be parsed.
      */
-    public Credentials validateTicket(String ticket) throws GuacamoleException {
+    public String validateTicket(String ticket, Credentials credentials) throws GuacamoleException {
```

Comment:
The new `credentials` parameter needs to be documented, too.

Reply:
Oops...added.